.github/workflows/build.yaml

```diff
@@ -18,43 +17,49 @@ on:
       sha:
         required: true
         type: string
-      build-type:
+      build_type:
         type: string
         default: nightly
+      wheel_versioneer_override:
+        type: string
+        default: ''
+      wheel_build_tag:
+        type: string
+        default: ''
 
 concurrency:
-  group: "ucx-py-${{ github.workflow }}-${{ github.ref }}"
+  group: ${{ github.workflow }}-${{ github.ref }}
```

Comment:
Hmm I didn't notice this difference in the other PRs, but here because wheels.yaml->build.yaml is treated like a file rename I see that the hardcoding is removed from the group. IIRC we originally added this because using the same shared action from different repos resulted in them all being part of the same concurrency group and therefore canceling each other, right? Is there a reason that isn't necessary any more? I assume it's working correctly or we'd be seeing conda CI from different repos conflicting all over the place, I'm just surprised.

Reply:
Our org use of GitHub Actions has evolved since then, with the following improvements:
(@ajschmidt8 gave me this info through slack):
> solved by adding the nightly- prefix to the top-level pipeline as opposed to the lower-level pipelines
https://github.com/rapidsai/actions/blob/main/.github/workflows/nightly-pipeline.yaml#L12
  group: nightly-${{ github.workflow }}-${{ github.ref }}

>however, this isn't needed anymore since we've switched to using workflow_dispatch events instead of workflow_call for the repo pipelines via the trigger-workflow-and-wait composite action that we chatted about yesterday
https://github.com/rapidsai/actions/blob/main/.github/workflows/nightly-pipeline.yaml#L35

Reply:
This change is nice!